app/export.go

```diff
@@ -103,7 +131,7 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, opts
 	}
 
 	ctx.Logger().Info("Bulk export: exporting emoji")
-	emojiPaths, err := a.exportCustomEmoji(writer, outPath, "exported_emoji", !opts.CreateArchive)
+	emojiList, err := a.exportCustomEmoji(writer, outPath, "exported_emoji", !opts.CreateArchive)
```

Comment:
any reason for this change

Reply:
Because, now we are not only taking path but also taking whole information of that emoji